tests/session.test.ts

```diff
@@ -525,7 +525,8 @@ test.describe('Session', () => {
 		await expect(page.getByText('Sorry, something went wrong.')).toBeVisible();
 		await expect(
 			page.locator('code', {
-				hasText: 'Error: Did not receive done or success response in stream.'
+				hasText:
+					"SyntaxError: Expected property name or '}' in JSON at position 2 (line 1 column 3)"
 			})
 		).toBeVisible();
```

Comment:
We no longer get this error which means that if the JSON syntax breaks we are back to the [old error](https://github.com/fmaclen/hollama/issues/103).

The main issue in #103 was that we were using the `/api/generate` endpoint which broke JSON syntax upon reaching the model's context window, that should no longer be the case 🤞 .